pkg/authorization/api_test.go

```diff
@@ -11,13 +11,13 @@ import (
 	"testing"
 	"time"
 
-	"gotest.tools/assert"
-	is "gotest.tools/assert/cmp"
+	"gotest.tools/v3/assert"
+	is "gotest.tools/v3/assert/cmp"
 )
 
 func TestPeerCertificateMarshalJSON(t *testing.T) {
 	template := &x509.Certificate{
-		IsCA:                  true,
+		IsCA: true,
```

Comment:
Why? 
Seems causing goimports error: https://github.com/moby/moby/pull/40501

Reply:
Ugh, sorry about that; I suspect this has been my IDE formatting it (perhaps different go version?)

Also curious why CI didn't catch it